src/test/scala/cromwell/MainSpec.scala

```diff
@@ -9,17 +9,16 @@ import akka.util.Timeout
 import better.files._
 import cromwell.util.FileUtil._
 import cromwell.util.SampleWdl
-import cromwell.util.SampleWdl._
+import cromwell.util.SampleWdl.{EmptyWorkflow, GoodbyeWorld, ThreeStep}
 import org.apache.commons.io.output.TeeOutputStream
 import org.scalatest.concurrent.TimeLimitedTests
 import org.scalatest.time.Span
-import org.scalatest.{BeforeAndAfterAll, FlatSpec, Matchers}
+import org.scalatest.{Matchers, BeforeAndAfterAll, FlatSpec}
```

Comment:
This is a really bizarre reordering. 

Also beware intellij auto-imports if that's what caused this. They can't handle implicits in certain cases.